app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= render "devise/shared/welcome_bar" %>
```

Comment:
since you are adding this line in so many files, is there a way you could make it a partial or something reusable instead? or is there a reason why you would have to do this <%= render "devise/shared/welcome_bar" %> in so many different files?

Reply:
It actually is a partial! welcome_bar is a partial that I'm then rendering on all of the pages. There's not really a more reusable way to do this

Reply:
oh my bad. nice work!